src/core/hle/service/pctl/module.cpp

```diff
@@ -119,92 +133,273 @@ class IParentalControlService final : public ServiceFramework<IParentalControlSe
     }
 
 private:
+    bool CheckFreeCommunicationPermissionImpl() {
```

Comment:
```suggestion
    bool CheckFreeCommunicationPermissionImpl() const {
```

Also should this function be returning true on all paths, or is that resolved along with the completion of the TODO comment? If so, it's probably best to explicitly indicate in a comment that returning by true on all paths is currently intended and not a bug.